platform-bungeecord/src/main/java/net/kyori/adventure/platform/bungeecord/BungeeFacet.java

```diff
@@ -255,7 +255,7 @@ public void bossBarInitialized(final net.kyori.adventure.bossbar.@NotNull BossBa
     @Override
     public void bossBarNameChanged(final net.kyori.adventure.bossbar.@NotNull BossBar bar, final @NotNull Component oldName, final @NotNull Component newName) {
       if (!this.viewers.isEmpty()) {
-        this.bar.setTitle(ComponentSerializer.toString(this.createMessage(this.viewers.iterator().next(), newName)));
+        this.bar.setTitle(TextComponent.fromArray(this.createMessage(this.viewers.iterator().next(), newName)));
```

Comment:
Few things need to change here:
1. We should be checking what method exists, and invoking either so it can continue to work on older versions.
2. We should be using the bungee serializer to turn our components into a bungee chat one, rather than cramming legacy into bungee components.

Reply:
>     2. We should be using the bungee serializer to turn our components into a bungee chat one, rather than cramming legacy into bungee components.

I'll work on that. I completely overlooked backward compatibility. Could you provide an example of how to use the Bungee serializer?